tests/static-analysis/Result/result_interface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Tests\StaticAnalysis\Iter;
+
+use Exception;
+use Psl\Result\Failure;
+use Psl\Result\ResultInterface;
+use Psl\Result\Success;
+
+/**
+ * @param ResultInterface<int> $result
+ *
+ * @throws Exception
+ *
+ * @return Failure<int, Exception>
```

Comment:
Stange : Shouldn't psalm complain about the second generic parameter `Exception` that does not exist here?